test/ecs/components/VirtualCamera.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { Engine, components } from '../../../packages/@dcl/ecs/src'
+import { testComponentSerialization } from './assertion'
+
+describe('VirtualCamera component', () => {
+  it('should serialize with time transition', () => {
+    const newEngine = Engine()
+    testComponentSerialization(components.VirtualCamera(newEngine), {
+      defaultTransition: {
+        transitionMode: {
+          $case: 'time',
+          time: 3
+        },
+        fromEntity: 6,
+        toEntity: 66
+      },
+      lookAtEntity: 35
+    })
+  })
+
+  it('should serialize with speed transition', () => {
+    const newEngine = Engine()
+    testComponentSerialization(components.VirtualCamera(newEngine), {
```

Comment:
maybe we can add some helper like we do for the tweens to avoid this `$case`.
```ts
VirtualCamera.creat(entity, {
  defaultTransition: {
    transitionMode: TranstionSpeed({ speed: 13 }),
    fromEntity: 9,
    toEntity: 99
  },
  lookAtEntity: 86
})

```

Reply:
Tackled at https://github.com/decentraland/js-sdk-toolchain/pull/990/commits/871c0094f9221d834cdd46df09ba6a7cc3bc8c70